<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_2051026490.jpg" id="_10_5_622020d_1132928309471_735853_6141"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_1555793073.jpg" name="UnrecoverableEntryException" refid="_10_5_622020d_1132928309461_267876_6139"/></annotatedElement><body humanName="Body" mode="s,e">This exception is thrown if an entry in the keystore cannot be recovered.

@version 1.2, 12/19/03

@since 1.5
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_1555793073.jpg" mode="s,e" name="UnrecoverableEntryException" refid="_10_5_622020d_1132928309461_267876_6139"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_1555793073.jpg" name="UnrecoverableEntryException" refid="_10_5_622020d_1132928309461_267876_6139"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>